resources/services/servicebus/namespaces.go

```diff
@@ -66,166 +66,373 @@ func ServicebusNamespaces() *schema.Table {
 				Type:        schema.TypeString,
 				Resolver:    schema.PathResolver("SBNamespaceProperties.MetricID"),
 			},
 			{
 				Name:        "zone_redundant",
-				Description: "Enabling this property creates a Premium Service Bus Namespace in regions supported availability zones.",
+				Description: "Enabling this property creates a Premium Service Bus Namespace in regions supported availability zones",
 				Type:        schema.TypeBool,
 				Resolver:    schema.PathResolver("SBNamespaceProperties.ZoneRedundant"),
 			},
 			{
 				Name:          "key_vault_properties",
-				Description:   "Properties of KeyVault (BYOK Encryption).",
+				Description:   "Properties of KeyVault",
 				Type:          schema.TypeJSON,
-				Resolver:      resolveServicebusNamespacesKeyVaultProperties,
+				Resolver:      schema.PathResolver("SBNamespaceProperties.Encryption.KeyVaultProperties"),
 				IgnoreInTests: true,
 			},
 			{
 				Name:        "key_source",
-				Description: "Enumerates the possible value of keySource for Encryption.",
+				Description: "Enumerates the possible value of keySource for Encryption",
 				Type:        schema.TypeString,
 				Resolver:    schema.PathResolver("SBNamespaceProperties.Encryption.KeySource"),
 			},
 			{
 				Name:          "require_infrastructure_encryption",
-				Description:   "Enable Infrastructure Encryption (Double Encryption).",
+				Description:   "Enable Infrastructure Encryption (Double Encryption)",
 				Type:          schema.TypeBool,
 				Resolver:      schema.PathResolver("SBNamespaceProperties.Encryption.RequireInfrastructureEncryption"),
 				IgnoreInTests: true,
 			},
 			{
 				Name:        "disable_local_auth",
-				Description: "This property disables SAS authentication for the Service Bus namespace.",
+				Description: "This property disables SAS authentication for the Service Bus namespace",
 				Type:        schema.TypeBool,
 				Resolver:    schema.PathResolver("SBNamespaceProperties.DisableLocalAuth"),
 			},
+			{
+				Name:        "location",
+				Description: "The Geo-location where the resource lives",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "tags",
+				Description: "Resource tags",
+				Type:        schema.TypeJSON,
+			},
+			{
+				Name:        "id",
+				Description: "Resource Id",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ID"),
+			},
+			{
+				Name:        "name",
+				Description: "Resource name",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "type",
+				Description: "Resource type",
+				Type:        schema.TypeString,
+			},
 		},
 		Relations: []*schema.Table{
 			{
 				Name:          "azure_servicebus_namespace_private_endpoint_connections",
-				Description:   "List of private endpoint connections.",
-				Resolver:      fetchServicebusNamespacesPrivateEndpointConnections,
+				Description:   "PrivateEndpointConnection properties of the PrivateEndpointConnection",
```

Comment:
This description seems a little strange; it might be worth overriding it in the generator config? I'm not sure, but maybe something like `Properties of the PrivateEndpointConnection on Servicebus Namespaces`